sensor/README.md

```diff
@@ -1,103 +1,385 @@
+At Tier IV we currently run several projects that use several types of sensors. Due to this, we do not create calibrators for each particular project, and instead reuse the same code, only modifying the parameters and helper nodes. For this reason, in the sensor calibration manager, we introduce the concepts of projects and calibrators. In this context, a project consists of a list of calibrators, with the same calibrator (semantically) being able to belong to multiple projects.
 
-- [Camera calibration via camera-lidar calibration](docs/how_to_extrinsic_interactive.md)
+An example of this structure is as follows:
 
-Traditionally, camera calibration is performed by detecting several views of planar boards in an image, and so, by knowing the board's dimensions, the camera matrix and distortion can be computed. However, this same process can also be performed by obtaining generic camera-object pairs, like the ones obtained during camera-lidar calibration. By reutilizing the calibration points from camera-lidar calibration, we can perform camera intrinsic calibration and camera-lidar extrinsic calibration simultaneously.
+```text
+calibrators/
+calibrators/projectA/
+calibrators/projectA/calibratorA.py
+calibrators/projectA/calibratorB.py
+calibrators/projectB/
+calibrators/projectB/calibratorA.py
+calibrators/projectB/calibratorB.py
+calibrators/projectC/
+calibrators/projectC/calibratorC.py
+```
+
+#### Calibrator interface
+
+A calibrator interface is the representation of the calibration process inside the sensor calibration manager. It specifies its project, the calibrator name, the `tf` that are required during the calibration process, and the expected frames that the calibrator node should return.
+
+Following the previous example, the `calibratorA.py` could be implemented as follows:
+
+```python
+@CalibratorRegistry.register_calibrator(
+    project_name="projectA", calibrator_name="calibratorA"
+)
+class CalibratorA(CalibratorBase):
+    required_frames = ["calibration_parent_frame", "calibration_child_frame", "auxiliar_frame"]
+
+    def __init__(self, ros_interface: RosInterface, **kwargs):
+        super().__init__(ros_interface)
+
+        self.add_calibrator(
+            service_name="the_name_of_the_calibration_service",
+            expected_calibration_frames=[
+                FramePair(parent="calibration_parent_frame", child="calibration_child_frame"),
+            ],
+        )
+
+```
+
+In addition to specifying required frames and calibration services names, the calibrator interfaces are also to post-process the calibration results if needed to accomodate to robotic frame conventions and other project-specific requirements.
```

Comment:
also **used** to ?

Reply:
Thanks. Addressed in [6ef4618](https://github.com/tier4/CalibrationTools/pull/164/commits/6ef4618077a75c54df34216cc0ef3da0f7cef01d)